glide.yaml

```diff
@@ -1,7 +1,5 @@
 package: k8s.io/helm
 import:
-- package: golang.org/x/text/
-  version: >=0.3.3
```

Comment:
We need to separate changes made to glide.yml and the corresponding vendor changes or else it's too difficult to review these changes.

Reply:
Ok, since you are both wanting this, I'll make a new PR without even touching the glide.yaml. The glide.yaml is autogend:
https://glide.readthedocs.io/en/latest/commands/

Reply:
New PR
https://github.com/kube-reporting/helm/pull/41
Closing this one.